Package.swift

```diff
@@ -270,7 +270,21 @@ let storageTargets: [Target] = [
     ),
 ]
 
-let targets: [Target] = amplifyTargets + apiTargets + authTargets + dataStoreTargets + storageTargets
+let analyticsTargets: [Target] = [
+    .target(
+        name: "AWSPinpointAnalyticsPlugin",
+        dependencies: [
+            .target(name: "Amplify"),
+            .target(name: "AWSCognitoAuthPlugin"),
+            .product(name: "AWSPinpoint", package: "AWSSwiftSDK")],
+        path: "AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin",
```

Comment:
are you excluding test target for now?

Reply:
I figured something was missing :P Yeah, we can add it later